### ISSUE TITLE
pain at the base of the thumb

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10704]
**Preferred term label:**
pain at the base of the thumb

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pain localized to the base of the thumb, over thenar eminence or thumb metacarpal

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Articular fracture of the base of the thumb metacarpal
De Quervain tenosynovitis
Basilar thumb arthritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001127
- label: Pain at the base of the thumb
- parents: [Hand pain (HP:0046505)](https://hpo.jax.org/app/browse/term/HP:0046505)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort localized to the base of the thumb, over thenar eminence or thumb metacarpal.
- comment: The base of the thumb is the fleshy area on the palm side of the hand near where the thumb attaches to the hand.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10704
- synonyms:
